x-pack/test_serverless/functional/test_suites/observability/dataset_quality/dataset_quality_details_degraded_field_flyout.ts

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import {
+  createDegradedFieldsRecord,
+  datasetNames,
+  defaultNamespace,
+  getInitialTestLogs,
+  ANOTHER_1024_CHARS,
+  MORE_THAN_1024_CHARS,
+} from './data';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const PageObjects = getPageObjects([
+    'common',
+    'navigationalSearch',
+    'observabilityLogsExplorer',
+    'datasetQuality',
+    'svlCommonPage',
+  ]);
+  const testSubjects = getService('testSubjects');
+  const synthtrace = getService('svlLogsSynthtraceClient');
+  const retry = getService('retry');
+  const to = '2024-01-01T12:00:00.000Z';
+  const degradedDatasetName = datasetNames[2];
+  const degradedDataStreamName = `logs-${degradedDatasetName}-${defaultNamespace}`;
+
+  describe('Degraded fields flyout', () => {
+    before(async () => {
+      await synthtrace.index([
+        // Ingest basic logs
+        getInitialTestLogs({ to, count: 4 }),
+        // Ingest Degraded Logs
+        createDegradedFieldsRecord({
+          to: new Date().toISOString(),
+          count: 2,
+          dataset: degradedDatasetName,
+        }),
+      ]);
+      await PageObjects.svlCommonPage.loginWithPrivilegedRole();
+    });
+
+    after(async () => {
+      // await synthtrace.clean();
+    });
+
+    describe('degraded field flyout open-close', () => {
+      it('should open the flyout when user clicks on the expand button', async () => {
+        await PageObjects.datasetQuality.navigateToDetails({
+          dataStream: degradedDataStreamName,
+        });
+
+        await PageObjects.datasetQuality.openDegradedFieldFlyout('test_field');
+
+        await testSubjects.existOrFail(
+          PageObjects.datasetQuality.testSubjectSelectors.datasetQualityDetailsDegradedFieldFlyout
+        );
+      });
+
+      it('should close the flyout when user clicks on close flyout button', async () => {
```

Comment:
Could we merge this and the previous one? `open and close` since this is what is happening for this test